teslajsonpy/controller.py

```diff
@@ -152,9 +152,6 @@ def valid_result(result):
             _LOGGER.debug(
                 "Exception: %s\n%s(%s %s)", str(ex), wrapped.__name__, args, kwargs
             )
```

Comment:
Please note 408 is quite common for Tesla as a response. We had bug reports on that and that's why I think we added this special handling. We need to consider the impact this will have.

Reply:
Yes I encounter a lot of 408s myself. The handling for 408 hasn't been removed, that's handled in the Controller itself, not in this wake_up wrapper.
This change just removes the setting of the state to offline when encountering a 408...this was causing a lot of flickering because tesla's api would say `online`, a transient 408 would set it to `offline`, then the `get_vehicles` would set it to `online`, etc.

I removed all of the state changes that weren't based on what was being informed from the Tesla API.

The next PR I'd like to add some retry logic to all calls to combat the transient 408s, and then if we see enough consecutive failures we can set a new state `unavailable` which would override anything that Tesla reports (until we start getting a success response again)

Reply:
I went ahead and implemented the `unavailable` state to show when we have consecutive tesla exceptions so it's more obvious to the user.

Reply:
Ok I can live with that.